account-kit/signer/src/client/base.ts

```diff
@@ -57,6 +59,15 @@ export abstract class BaseSignerClient<TExportWalletParams = unknown> {
     );
   }
 
+  /**
+   * Asynchronously fetches and sets the OAuth configuration.
+   *
+   * @returns {Promise<OauthConfig>} A promise that resolves to the OAuth configuration
+   */ public initOauth = async (): Promise<OauthConfig> => {
```

Comment:
sometimes the linter does this and I haven't tracked down why

```suggestion
   */ 
   public initOauth = async (): Promise<OauthConfig> => {
```